src/overlays/actors/ovl_Bg_Hidan_Firewall/z_bg_hidan_firewall.c

```diff
@@ -21,23 +22,239 @@ const ActorInit Bg_Hidan_Firewall_InitVars =
     (ActorFunc)BgHidanFirewall_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Hidan_Firewall/BgHidanFirewall_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Hidan_Firewall/BgHidanFirewall_Destroy.s")
+static ColliderCylinderInit cylinderInitData =
+{
+    0x0A, 0x11, 0x00, 0x09,
+    0x20, 0x01, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00,
+    0x20000000,
+    0x01, 0x04, 0x00, 0x00,
+    0xFFCFFFFF,
+    0x00, 0x00, 0x00, 0x00,
+    0x19, 0x00, 0x01, 0x00,
+    0x001E,
+    0x0053,
+    0x00000000000000000000,
+};
+
+static SubActor98Init actor98InitData =
+{
```

Comment:
this struct doesnt exist anymore, we have specific structs for 4 vs 5 init variables. Make sure you merge with master before doing a PR so we catch stuff like this